doc/installation.rst

```diff
@@ -18,54 +18,57 @@ Begin by installing the conda package manager.  If conda is already installed sk
+In principle, any Conda distribution will work, but it is probably best to install `Mambaforge <https://github.com/conda-forge/miniforge#mambaforge>`_ to a local directory and to avoid running `conda init`. After performing a local directory installation, Mambaforge can be activated via the [scripts,bin]/activate scripts. This will help avoid polluting and breaking the local Python installation.
+
+Mambaforge is a minimal installer that sets *conda-forge* as the default channel for package installation and provides **mamba**.   ``mamba install`` is recommended over ``conda install`` for faster and less memory intensive cadquery installation.
 
 In Linux/MacOS, the local directory installation method looks something like this::
 
-    # Install the script to ~/miniforge
-    wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O miniforge.sh
-    bash miniforge.sh -b -p $HOME/miniforge
+    # Install to ~/mambaforge
+    curl -L -o mambaforge.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh"
+    bash mambaforge.sh -b -p $HOME/mambaforge
 
-    # To activate and use Miniconda
-    source $HOME/miniforge/bin/activate
+    # Activate
+    source $HOME/mambaforge/bin/activate
 
 
 On Windows, download the installer and double click it on the file browser or install non-interactively as follows::
 
-    :: Install
-    curl -L -o miniforge.exe https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
-    start /wait "" miniforge.exe /InstallationType=JustMe /RegisterPython=0 /NoRegistry=1 /NoScripts=1 /S /D=%USERPROFILE%\Miniforge3
+    :: Install to %USERPROFILE%\Mambaforge
+    curl -L -o mambaforge.exe https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Windows-x86_64.exe
+    start /wait "" mambaforge.exe /InstallationType=JustMe /RegisterPython=0 /NoRegistry=1 /NoScripts=1 /S /D=%USERPROFILE%\Mambaforge
 
     :: Activate
-    cmd /K ""%USERPROFILE%/Miniforge3/Scripts/activate.bat" "%USERPROFILE%/Miniforge3""
+    cmd /K ""%USERPROFILE%/Mambaforge/Scripts/activate.bat" "%USERPROFILE%/Mambaforge""
 
 It might be worthwhile to consider using ``/NoScripts=0`` to have an activation shortcut added to the start menu.
 
 After conda installation, create and activate a new `conda environment <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_ to prepare for cadquery installation.
 
-Note that miniforge automatically sets *conda-forge* as the default channel.  ``-c conda-forge`` can be omitted from the install command when using a miniforge installation.
-
 
 conda
 `````
 
-Install the latest released version of cadquery [#f1]_::
+Install the latest released version of cadquery::
 
-    conda create -n cqrel
-    conda activate cqrel
-    conda install -c conda-forge cadquery occt=7.7.0
```

Comment:
I like this PR and the improvements it makes to thin install docs. Just one question do you think removing the ```-c conda-forge``` part will trip up some conda users who are not using mambaforge or miniforge. I'm assuming some people are going to copy paste these instructions. Would it be worth keeping this in which would still work for all?

Reply:
AFAICT  channel spec is still there

Reply:
The installation guide discusses the default channel where the README does not so I thought I'd leave the explicit channel '-c conda-forge' in the README command.